rotkehlchen/assets/converters.py

```diff
@@ -1069,3 +1069,21 @@ def asset_from_cryptocom(cryptocom_name: str) -> Asset:
 
     symbol = CRYPTOCOM_TO_WORLD.get(cryptocom_name, cryptocom_name)
     return symbol_to_asset_or_token(symbol)
+
+
+LOCATION_TO_ASSET_MAPPING: Dict[Location, Callable] = {
```

Comment:
More exact typing. Specify exact callable args

Also ... hmm ... there must be a better way to do this but can't say now off the top of my head